### ISSUE TITLE
Blank line after table header discards header syntax

### ISSUE THREAD

**Issue by codesniffer13** [doxygen/doxygen#7409]
Seems that if a markdown table has a blank line after a table header cell, the header is not rendered specially.

Seeing this with Doxygen 1.8.16, Windows & Linux.

Repro Example:

```
<table>
<tr>
	<th>Column 1
	<th>Column 2

<tr>
	<td>Data 1
	<td>Data 2
</table>
```

Notice how "Column 2" is rendered compared to "Column 1".  If the blank line after "Column 2" is removed it's rendered the same.

**Comment by codesniffer13** [comment]
Incidentally, looks like GitHub accepts the same markdown and renders the Repro Example as I expected:

<table>
<tr>
	<th>Column 1
	<th>Column 2

<tr>
	<td>Data 1
	<td>Data 2
</table>

**Comment by albert-github** [comment]
Always and especially here as the output depends on an empty line / spacing (and this is sometimes removed during the display):
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem?  Please don't add external links as they might not be persistent.

I see the return as well and I think it is correct the `<th>` is not explicitly closed but implicitly closed by the `<tr>` (also the `<tr>` is implicitly closed).

I see that the "Column 1" is bold but the "Column  2" isn't, so ths is something to look at. Looks like the empty line anyway plays a special role in this case.

**Comment by albert-github** [comment]
Regarding the problem with the non bold of "Column 2", I've just pushed a proposed patch, pull request #7414

**Comment by albert-github** [comment]
Code regarding #7414 has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.17.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
